5_databases/week1/homework

```diff
@@ -0,0 +1,55 @@
+-- Active: 1699216915341@@0.0.0.0@3306@hyf_lesson1
+
+SELECT COUNT(*)
+ FROM task;
+
+
+
+
+SELECT COUNT(*) 
+FROM task 
+WHERE due_date <= CURDATE();
+
+
+
+SELECT *
+FROM task
+WHERE status_id= 3;
+
+
+SELECT *
+FROM task
+WHERE status_id!= 3;
+
+
+
+SELECT *
+FROM task
+ORDER BY created DESC;
+
+
+SELECT MAX(created)
+from task;
+
+SELECT title, due_date
```

Comment:
Good start, but we miss to search for the "database" string in title and description fields.

You do this with the `LIKE` operator:
https://www.w3schools.com/mysql/mysql_like.asp